pychromecast/controllers/spotify.py

```diff
@@ -11,10 +11,9 @@
 APP_NAMESPACE = "urn:x-cast:com.spotify.chromecast.secure.v1"
 TYPE_GET_INFO = "getInfo"
 TYPE_GET_INFO_RESPONSE = "getInfoResponse"
-TYPE_SET_CREDENTIALS = "setCredentials"
```

Comment:
Why is this removed?

Reply:
Based on the net-export log, our web player wasn't sending this anymore. We tried it with the same auth token, and it still returned `setCredentialsError`. Doing the same `addUser` the web player was sending worked.